documentation/docs/00-getting-started.md

```diff
@@ -2,37 +2,17 @@
 
 ## Installation from Unity Package
 
-1. Visit https://www.mapbox.com/unity/
-
-2. Click `Download the SDK`
-
-3. If you already have a Mapbox SDK imported in your project, please delete the existing `Mapbox` folder to avoid potential conflicts
-
-4. Import the package in Unity
-
-   - Assets—>Import package—>Custom Package (from the Unity editor menu) 
-
-    **OR**
-
-   - Double-click downloaded `.unityPackage`
+Check out the official [Unity SDK page](https://www.mapbox.com/unity-sdk/) for installation steps.
 
 ## Installation from Git (source)
 
 1. `git clone https://github.com/mapbox/mapbox-unity-sdk.git`
 2. [Read about how the project is configured](https://github.com/mapbox/mapbox-unity-sdk/blob/develop/README.md) and [learn how to contribute](https://github.com/mapbox/mapbox-unity-sdk/blob/develop/CONTRIBUTING.md)
```

Comment:
IMO, there should be instructions to run the `update` script here instead of sending the user to the `Readme`

Reply:
What's the `update` script? I'm not sure I've used it before.

By the way, the rationale for removing some of this installation content is because we already have instructions at /unity-sdk. Should we update those instructions to mention this script?

Reply:
This is the script I am talking about `update-mapbox-unity-sdk-core` , instructions are here https://github.com/mapbox/mapbox-unity-sdk/blob/develop/README.md#from-this-repository-advanced-latest-development

This script is required to fetch the dependencies for the SDK. Without running the scripts the SDK will not work. 

The instructions on the /unity-sdk page are about getting the package & released versions of the SDK. If someone is cloning the repo - these instructions would be handy. I don't mind having a duplicate of just `Installation from Git` in both places.

Reply:
Ohh got it got it. That's a good idea! I'll update the link.